src/modules/inventory/index.ts

```diff
@@ -1,4 +1,4 @@
-import { ClientType } from '../../index';
+import { ClientType, GET_ALL_PRODUCT } from '../../index';
```

Comment:
```suggestion
import { ClientType, GET_ALL_PRODUCTS } from '../../index';
```

Please update all corresponding references.